### PR TITLE
Issue #129

### PR DESCRIPTION
**Pull request by kirjaamo**
New parameters added.

### CODE REVIEW

File: galeracluster/source/documentation/galera-parameters.rst [hunk 1]
```diff
@@ -278,6 +280,29 @@ on the master are committed.
...
+
+   "Default Value", "``/var/lib/mysql/``"
+   "Dynamic", "Yes"
+   "Initial Version", "1.0"
+
+The excerpt below is an example of how this Galera parameter might look in the configuration file:
+
+.. code-block:: ini
+
+   wsrep_provider_options="datadir=/var/lib/mysql/"
```
Comment (byte) [inline galeracluster/source/documentation/galera-parameters.rst:303]: er? where did you get this from? you can't place the datadir in wsrep_provider_options, it is in my.cnf by titself

File: galeracluster/source/documentation/galera-parameters.rst [hunk 2]
```diff
@@ -1476,6 +1501,47 @@ This status variable is used to check which gmcast protocol version is used.
...
+
+The possible values are:
+
+- ``0`` - Nothing is done on commit; rather the log buffer is written and flushed to the InnoDB redo log once a second. This gives better performance, but a server crash can erase the last second of transactions.
+- ``1`` - The log buffer is written to the InnoDB redo log file, and a flush to disk performed after each transaction. After a crash, committed transactions will not be lost and will be consistent with the binlog and other transactional engines. This is required for full ACID compliance.
+- ``2`` - The log buffer is written to the InnoDB redo log after each commit, but flushing takes place once a second. With this option, your system can get inconsistent and lose the last second's transactions if there is a power failure or kernel crash, but not if ``mysqld`` crashes.
+- ``3`` - Flush to disk at prepare and at commit. This option is slower and usually redundant. After a crash, committed transactions will not be lost and will be consistent with the binlog and other transactional engines. With this option, InnoDB emulates the older implementation of group commit, with 3 syncs to disk per group commit.
+
```
Comment (byte) [inline galeracluster/source/documentation/galera-parameters.rst:1519]: Need you to make sure you're looking at 8.0/8.4 and also comparing well with mariadb (which has some changes - https://mariadb.com/kb/en/innodb-system-variables/#innodb_flush_log_at_trx_commit)

File: galeracluster/source/documentation/galera-parameters.rst [hunk 3]
```diff
@@ -2352,6 +2418,30 @@ The excerpt below is an example of how this Galera parameter can be triggered fr
...
+   "Default Value", "0"
+   "Dynamic", "Yes"
+   "Initial Version", ""
+
+The excerpt below is an example of how this Galera parameter might look in the configuration file, using the maximum value of the parameter:
+
+.. code-block:: ini
+
+   SET GLOBAL wsrep_provider_options = 'sync_binlog=4294967295';
```
Comment (byte) [inline galeracluster/source/documentation/galera-parameters.rst:2441]: no, sync_binlog is not a wsrep_provider_option 

https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_sync_binlog

**Review by byte — CHANGES_REQUESTED** [review]
changes needed

File: galeracluster/source/documentation/galera-parameters.rst (hunk 1 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/galera-parameters.rst]: Fixed

File: galeracluster/source/documentation/galera-parameters.rst (hunk 2 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/galera-parameters.rst:1519]: Checked and edited.

File: galeracluster/source/documentation/galera-parameters.rst (hunk 3 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/galera-parameters.rst]: Updated.

File: galeracluster/source/documentation/galera-parameters.rst [hunk 4]
```diff
@@ -2352,6 +2418,36 @@ The excerpt below is an example of how this Galera parameter can be triggered fr
...
 
 
+.. _`sync_binlog`:
+.. rst-class:: section-heading
+.. rubric:: ``sync_binlog``
+
+.. index::
+   pair: wsrep Provider Options; sync_binlog
+
+Synchronously flush binary log to disk after every #th event. The options are:
```
Comment (byte) [inline galeracluster/source/documentation/galera-parameters.rst:2428]: what is #th event ?

File: galeracluster/source/documentation/galera-parameters.rst [hunk 5]
```diff
@@ -1476,6 +1501,47 @@ This status variable is used to check which gmcast protocol version is used.
...
+
+   innodb_flush_log_at_trx_commit="1"
+
+This variable can also be set dynamically at runtime:
+
+.. code-block:: mysql
+
+   SET GLOBAL innodb_flush_log_at_trx_commit=1;
+
+If you set ``innodb_flush_log_at_trx_commit`` dynamically at runtime, its value will be reset the next time the server restarts. To make the value persist on restart, set it also in a configuration file.
```
Comment (byte) [inline galeracluster/source/documentation/galera-parameters.rst:1541]: Add a note that if you use MySQL 8 and greater, you can also use SET PERSIST to ensure the value persists upon restart.

**Review by byte — CHANGES_REQUESTED** [review]
minor changes requested

File: galeracluster/source/documentation/galera-parameters.rst (hunk 4 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/galera-parameters.rst:2428]: It means, for example, every tenth event.

File: galeracluster/source/documentation/galera-parameters.rst (hunk 5 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/galera-parameters.rst:1541]: Added

**Review by byte — APPROVED** [review]
ok

**Pull request merged**